Late submission completed all tasks

Sergo-Bushmakin76 opened the pull request:
works, just not sure if its the best way to do it.

Sergo-Bushmakin76 committed:
Late submission completed all tasks

Sergo-Bushmakin76 committed:
changed the third function return

Sergo-Bushmakin76 commented:
please let me know why this test is failing, i cant seem to understand the reason because when I run the program it seems to work fine and there's no lint error either.

jwokaty commented:
@Sergo-Bushmakin76 Your functions only works when degrees is an int; it should work when degrees is also a string. Look at the examples and you'll notice that sometimes the input is a string rather than an int. There was a hint in Task 1. Secondly, your calculation for Task 3 is not correct. Please review the task specifications.

Commit:
changed third function and added ability to use strings as input

Sergo-Bushmakin76 commented:
this time im really not sure whats wrong, I corrected the third function and added the option for string as input and its still giving me a fail on the check.  I do pylint and bash runtests with no issues coming up.

jwokaty commented:
@Sergo-Bushmakin76 I see that you converted degrees to an int in your section function and that got rid of one error, but that's not really how you should accomplish this task. You really need to go back to Task 1's hint: it asked you to carefully consider where the decimal conversation takes place. Understanding how the decimal module works and following the hint is key to the assignment.

Commit:
Hopefully all done!
